testData/solutions/win32.json

```diff
@@ -970,6 +970,41 @@
         ]
     },
 
+    "com.microsoft.windows.displaySettings" {
+        "name": "Windows Display Settings",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": [
+                "capabilities": [
+                    "display.screenEnhancement.displaySettings",
+                    "applications.com\\.microsoft\\.windows\\.displaySettings.id"
+                ]
+            ]
+        }
+        "configure": [
+            "settings.configure"
+        ],
+        "restore": [
+            "settings.configure"
+        ],
+        "start": [
+            {
+                "type": "gpii.windows.spiSettingsHandler.setScreenResolution"
```

Comment:
To answer my own questions - This should be similar to @javihernandez's handler for [xrandr on Linux](https://github.com/GPII/universal/pull/408/files) which answers much and here is [the documentation](https://github.com/GPII/universal/blob/master/documentation/SolutionsRegistryFormat.md)